API.md

```diff
@@ -113,6 +113,10 @@ Definitiontable:
 - `major_button` Major (highlighted) button image
 - `minor_button` Minor button image
 - `node_color` Palette number to set if the node is paramtype2 = "colorfacedir"
+- `taskbar_clock_position`
```

Comment:
taskbar_clock_position_and_size 

Pls. add short description what the setting does